app/client/components/helpCentre/helpCentreTopic.tsx

```diff
@@ -49,17 +48,19 @@ const HelpCentreTopic = (props: HelpCentreTopicProps) => {
       );
   }, [props.topicCode]);
 
-  const selectedNavTopic = helpCentreNavConfig.find(
-    topic => topic.id === props.topicCode
-  );
+  const setSelectedTopicObject = React.useContext(SelectedTopicObjectContext);
+  useEffect(() => {
+    const selectedNavTopic = helpCentreNavConfig.find(
+      topic => topic.id === props.topicCode
+    );
```

Comment:
As in Article I don't think you need this check anymore. You can pass `props.topicCode` directly on line 57 and that way not import `helpCentreNavConfig` on this file at all.